reworked docs and made component standards easier to follow/reference (#80)
AdaloHQ/docs: website/docs/component-standards/interactions/list-controls.md

---
id: list-controls
title: List Controls
---

## Disabling List Controls

> There are some situations where you should disable list controls, such as sorting and filtering. In general, any time that a list control either doesn't make sense in the context of your component or doesn't have any effect on how the component is rendered, then the control should be disabled.

Reviewer: ```suggestion
There are some situations where you should disable list controls, such as sorting and filtering. In general, if a list control either doesn't make sense in the context of your component or doesn't have any effect on how the component is rendered, the control should be disabled.
```

> There are some situations where you should disable list controls, such as sorting and filtering. In general, any time that a list control either doesn't make sense in the context of you're component or doesn't have any effect on how the component is rendered, then the control should be disabled.

Reviewer 2: ```suggestion
There are some situations where you should disable list controls, such as sorting and filtering. In general, any time that a list control either doesn't make sense in the context of your component or doesn't have any effect on how the component is rendered, then the control should be disabled.
```
